src/User.hpp

```diff
@@ -18,8 +19,13 @@ class User
     void setGroup(std::string group);
     void setGitHub(std::string gitHub);
     void setFirecode(std::string firecode);
+    
     std::string getNick() const;
 
+    const std::string& getName() const;
+
+    User& operator=(const User& user) = default;
```

Comment:
Why do we want to copy a user? I wouldn't like to do it :)

Reply:
User should be copyable because the UserContainer have ownership of the user and only via provided methods Update and delete user object should be changed. 

This mean that retrive user need to return copy of an object rather than orginal object.

Reply:
corrected